intro-HPC/ch_x2go.tex

```diff
@@ -155,3 +155,13 @@ \section{Resume a previous session}
 This will give you the full login name (like ``\strong{\emph{\loginhost{}}}'' but the hostname in your situation may be slightly different).
 You should set the same name to resume the  session the next time.
 Just add this full hostname into ``login hostname'' section in your X2Go session (see \autoref{subsec:sessions-x2go-optionb}).
+
+\section{Connection faild with X2Go}
+\label{sec:conn-fail-x2go}
+
+If you get the error ``\strong{Connection Failed.} session vscXXYYY-123-4567890123\_xyzXFCE\_dp32 terminated'' (or similar),
+It is a possibility that an old X2go session remained on the login node.
```

Comment:
`It is possible that`

`X2go` -> `X2Go`

Reply:
done